ras/ras.py

```diff
@@ -188,13 +192,15 @@ def test9_ofpathname(self):
                       "=====")
         self.run_cmd("ofpathname -h")
         self.run_cmd("ofpathname -V")
-        disk_name = process.system_output("df -h | egrep '(s|v)da[1-8]' | "
-                                          "tail -1 | cut -d' ' -f1",
-                                          shell=True).strip()
+        disk_name = self.run_cmd_out("df -h | egrep '(s|v)da[1-8]' |"
+                                     " tail -1 | cut -d' ' -f1").strip()
+        self.run_cmd("ofpathname -V")
+        disk_name = self.run_cmd_out("df -h | egrep '(s|v)da[1-8]' | "
+                                     "tail -1 | cut -d' ' -f1").strip()
         if disk_name:
             self.run_cmd("ofpathname %s" % disk_name)
-            of_name = process.system_output("ofpathname %s"
-                                            % disk_name).strip()
+            of_name = self.run_cmd_out("ofpathname %s"
+                                       % disk_name).strip()
```

Comment:
Is it better for `run_cmd_out()` to return the output after a `strip()` ?

Reply:
sure make sense will adopt it